Lib/test/test_print.py

```diff
@@ -155,6 +155,15 @@ def test_string_with_excessive_whitespace(self):
 
         self.assertIn('print("Hello World", end=" ")', str(context.exception))
 
+    def test_string_with_stream_redirection(self):
+        import sys
+        python2_print_str = 'print >> sys.stderr, "message"'
+        with self.assertRaises(TypeError) as context:
+            exec(python2_print_str)
+
+        self.assertIn('Did you mean "print(<message>, '
+                'file=<output_stream>)', str(context.exception))
+
```

Comment:
Hi @serhiy-storchaka 

Can you please explain a little about the above cases?

For 1, As far as I understand the `max` function when redirected to `sys.stderr` should not cause the `Did you mean ...` statement to be executed.

So, a test case can be 

```
    def test_stream_redirection_using_max(self):
        import sys
        python2_print_str = 'max >> sys.stderr'
        with self.assertRaises(TypeError) as context:
            exec(python2_print_str)

        self.assertNotIn('Did you mean "print(<message>, '
                'file=<output_stream>)', str(context.exception))
```

For the 2, I don't quite understand if we just want to test `print >>` with the current patch. In that case again the `Did you mean...` statement would not be executed.

Is this what we want to test here?

Please correct me.

Thanks!